api/src/main/resources/liquibase.xml

```diff
@@ -749,4 +749,14 @@
             update patient_appointment set date_appointment_scheduled = date_created where date_appointment_scheduled is null;
         </sql>
     </changeSet>
+    <changeSet author="aojwang" id="add_index_on_patient_appointment_start_date_time_202411281139">
+        <createIndex indexName="patient_appointment_start_date_time_index" tableName="patient_appointment">
+            <column name="start_date_time"/>
+        </createIndex>
+    </changeSet>
+    <changeSet author="aojwang" id="add_index_on_patient_appointment_status_202411281152">
+        <createIndex indexName="patient_appointment_status_index" tableName="patient_appointment">
+            <column name="status"/>
```

Comment:
I am not sure if a status index is helpful, since a huge number of appointments, over time, will have the same status. If index on date-time is already there, then the set of appointments is already narrowed to a small set, where selecting status should not be much work for DB while querying.

Can you validate if the speed up is worth it, with date_time vs (date_time+status)?

Reply:
I have conducted a few tests and your POV is accurate. I ran queries with and without the index on status and the execution time is the same.